README.Rmd

```diff
@@ -54,6 +54,18 @@ To execute Rust code, you will also need to set up a working Rust toolchain. See
 
 ## Usage
 
+The status of Rust toolchain and available targets can be checked with `rust_sitrep()`:
+
+```{r eval=FALSE, echo=TRUE}
```

Comment:
I think this part can be expanded by 1 or 2 sentences. Can we suggest this as the first step for new users to run to ensure that they have a compatible version of Rust installed. If not, the function will guide them on how to install the appropriate tool chain?